cacheops/invalidation.py

```diff
@@ -25,7 +33,8 @@ def invalidate_dict(model, obj_dict, using=DEFAULT_DB_ALIAS):
     prefix = get_prefix(_cond_dnfs=[(model._meta.db_table, list(obj_dict.items()))], dbs=[using])
     load_script('invalidate')(keys=[prefix], args=[
         model._meta.db_table,
-        json.dumps(obj_dict, default=str)
+        json.dumps(obj_dict, default=str),
+        conj_del_fn()
```

Comment:
What does calling this function here do?

Reply:
The first time it's called, it executes the function, which sends Redis the `INFO` command and returns `'unlink'` or `'del'` depending on the Redis version in the response. On subsequent calls the `memoize` wrapper just returns the same string and skips the Redis command.

Reply:
I have a PR to upstream [where I suggest](https://github.com/Suor/django-cacheops/issues/288#issuecomment-412894722) that if we don't want to incur the `memoize` lookup on each invalidation we can assign it to a module global instead.

Reply:
Should we document that? `// Call memoized conj_del_fn function so that it gets saved in memory` or something? I don't know maybe it's self-explanatory 🤷‍♀️

Reply:
The cacheops dev, suor, doesn't like passing it in on every call, so this is going to change anyway.